src/main/interface/settings.c

```diff
@@ -1167,6 +1167,7 @@ const clivalue_t valueTable[] = {
     { "cpu_overclock",              VAR_UINT8  | MASTER_VALUE | MODE_LOOKUP, .config.lookup = { TABLE_OVERCLOCK }, PG_SYSTEM_CONFIG, offsetof(systemConfig_t, cpu_overclock) },
 #endif
     { "pwr_on_arm_grace",           VAR_UINT8  | MASTER_VALUE, .config.minmax = { 0, 30 }, PG_SYSTEM_CONFIG, offsetof(systemConfig_t, powerOnArmingGraceTime) },
+    { "scheduler_optimize_rate",    VAR_UINT8  | MASTER_VALUE | MODE_LOOKUP, .config.lookup = { TABLE_OFF_ON }, PG_SYSTEM_CONFIG, offsetof(systemConfig_t, schedulerPolicy) },
```

Comment:
I think the way it is now is a bit confusing and prone to being broken in the future, as on / off is mapped into `schedulerPolicy_e` here. It would be better to change the parameter to be `scheduler_optimize_rate` as well, and then use a ternary in `schedulerSetPolicy(systemConfig()->scheduler_optimize_rate ? SCHEDULER_POLICY_PRIORITIZE_AVERAGE_RATE : SCHEDULER_POLICY_PRIORITIZE_PERIOD)`.

Reply:
@mikeller I just changed it all to optimizeRate now for consistency.